src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -431,3 +432,132 @@ const struct pcm_func_map pcm_func_map[] = {
 const size_t pcm_func_count = ARRAY_SIZE(pcm_func_map);
 
 #endif
+
+static int pcm_convert_s16_c16_to_s16_c32(const struct audio_stream *source,
```

Comment:
There are Kconfig settings for supported sample formats, this code should use use the Kconfig to only build the selected formats.

Reply:
sure